content/en/docs/tasks/security/ca-config/_index.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Is this directory still needed now, since there is only 1 task under it? Should we move the task up on level instead?

Reply:
Done.